app/webpack.config.js

```diff
@@ -26,10 +26,9 @@ module.exports = {
         ]),
     ],
     devServer: {
-        publicPath: '/assets/',
-        historyApiFallback: {
-            index: '/assets/'
-        }
+        contentBase: './static/main',
+        hot: true,
+
```

Comment:
Why this change? I'm asking **about each changed line here**. Maybe it's OK, but I feel it's not and I want to understand it.

Reply:
It's the content base to run the web development service, because all the files are moved to this path.

Reply:
Danilo, please answer my question above. I'm asking about each line chagned. Could you please describe every line changed with information why it was changed like this? Why `-publicPath` why `-historyApiFallback` why `+ contentBase` and why `+hot` ?